lib/MetaCPAN/Query/Release.pm

```diff
@@ -560,11 +560,30 @@ sub all_by_author {
 }
 
 sub versions {
-    my ( $self, $dist ) = @_;
+    my ( $self, $dist, $versions ) = @_;
 
     my $size = $dist eq 'perl' ? 1000 : 250;
+
+    my $query;
+
+    if ($versions) {
+        $query = {
+            filtered => {
```

Comment:
We should try to use `bool` queries rather than `filtered` queries.  `filtered` has been deprecated in newer ES versions.

Reply:
You are right, I've been away so long I started to forget stuff:)

Corrected now.